### ISSUE TITLE
[FEATURE REQUEST] Add build preview to pull request

### ISSUE THREAD

**Issue by darigovresearch** [ONEARMY/project-kamp-academy#10]
This has been done to the precious plastic academy and is very helpful for testing that a pull request is doing what it says it should be doing

**Comment by darigovresearch** [comment]
@chrismclarke was it you that set this up?

**Comment by chrismclarke** [comment]
Yep, agree always a handy feature to have. For the pp-academy I setup this with `netlify`, which is a pretty good option for something that just works quickly out of the box. 

I think anyone with repo access can do this by connecting their own (free) netlify account. Here's a copy of the settings I have on my personal netlify for the pp-academy (if I can remember correctly it also meant adding another siteConfig file which you can see is copied over, because github pages and netlify use slightly different base URL structures)

![image](https://user-images.githubusercontent.com/10515065/122683678-9a1fe780-d1b5-11eb-9ca4-39c44b9bee75.png)

I'm running low on free build minutes on my netlify account, but if someone else want to try then that would be great. Other options would probably either be `vercel` (works similar to netlify) or `firebase` (probably overkill)
